includes/class-wi-volunteer-management.php

```diff
@@ -81,6 +81,7 @@ public function __construct() {
 
 		// Load the public hooks
 		$this->define_public_hooks();
+      $this->define_widget_hooks();
```

Comment:
@jg314 

**Pull request updated**:

Per above comment, I've moved the register_function back into widget/class-widget.php and am referencing the widget class when calling the add_action method to initialize the widget within the define_public_hooks function.